alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/alerts/AlertIndices.kt

```diff
@@ -57,20 +63,24 @@ import org.elasticsearch.threadpool.ThreadPool
  */
 class AlertIndices(
     settings: Settings,
-    private val client: IndicesAdminClient,
+    private val client: Client,
```

Comment:
Why did we change this to normal client? I see you changed all `client` to `client.admin()` was it just to use the suspend until?

Reply:
With the changes to ```AlertIndices``` now also making requests for index metadata when deleting old history indices, it made sense to change this to ```Client```. That way we can use both ```IndicesAdminClient``` and ```ClusterAdminClient``` as necessary within the class, similar to how it is in ISM.